src/main.rs

```diff
@@ -0,0 +1,175 @@
+use typedb_client::{
+    concept::{Attribute, Concept, Value},
+    Connection, DatabaseManager, Session,
+    SessionType::{Data, Schema},
+    TransactionType::{Read, Write}
+};
+use futures::StreamExt;
+use futures::TryStreamExt;
+use  std::fs;
+use std::io;
+
+#[derive(Debug)]
+enum HandleError {
+    Io(io::Error),
+    TypeDB(typedb_client::error::Error),
+}
+
+const TEST_DATABASE: &str = "codeforces-data-model";
+ 
+fn new_core_connection() -> typedb_client::Result<Connection> {
+    Connection::new_plaintext("localhost:1729")
+}
+
+fn read_input() -> Result<String, HandleError>{
+    let mut input_string = String::new();
+    io::stdin().read_line(&mut input_string).map_err(HandleError::Io)?;
+    input_string.pop();
+    Ok(input_string)
+}
+
+async fn load_data(connection: Connection)->Result<(), HandleError>{
```

Comment:
Can we use a more consistent code formatting? For instance, the arrows between the function declaration and the return type have inconsistent spacing. We'd recommend you to use the [`rustfmt`](https://github.com/rust-lang/rustfmt) tool to format your code in a consistent way.